app/views/users/show.html.erb

```diff
@@ -2,19 +2,27 @@
   <%= gravatar_for @user %>
   <%= @user.name %>
 </h1>
-<% if @user.posts.any? %> 
+<% if @user.posts.any? %>
 	<h3> Posts (<%= @user.posts.count %>)</h3>
-	  <ol> 
-		  <% @user.posts.each do |post| %> 
-        <li> 
-          <%= link_to post.title, post.url %> 
+	  <ol>
+		  <% @user.posts.each do |post| %>
+        <li>
+          <%= link_to post.title, post.url %>
           <br>
           <br>
           Posted <%= time_ago_in_words(post.created_at) %> ago.
         </li>
```

Comment:
Woops, I didn't even realized that I pushed this up before editing out bad spacing later on, because I couldn't even find that error in my current project. Thanks for the heads up.